### ISSUE TITLE
Create set PingInterval helper function

### ISSUE THREAD

**Issue by karpovicham** [nats-io/go-nats#359]
These are lots of helper functions to set connection options (which is great), but not for `PingInterval` 
https://github.com/nats-io/go-nats/blob/master/nats.go#L202

Could we please have such function for this option as well, i.e.: 
```
// PingInterval is an Option to set the period for client ping commands
func PingInterval(t time.Duration) Option {
	return func(o *Options) error {
		o.PingInterval = t
		return nil
	}
}
```

Something similar to `ReconnectWait` function:
https://github.com/nats-io/go-nats/blob/master/nats.go#L534

Thanks.

**Comment by derekcollison** [comment]
Feel free to submit a PR with tests etc. Otherwise we will leave this open and get to it as appropriate.

**Comment by derekcollison** [comment]
Resolved via #361
